src/NUnitFramework/tests/Assertions/LowTrustFixture.cs

```diff
@@ -61,7 +61,12 @@ public void AssertEqualityInLowTrustSandBox()
             });
         }
 
-        [Test, Platform(Exclude="Mono,MonoTouch", Reason= "Mono does not implement Code Access Security")]
+        //[Test, Platform(Exclude="Mono,MonoTouch", Reason= "Mono does not implement Code Access Security")]
+        // TODO: This test fails to run correctly when Assert.Throws
+        // attempts to create an isolated TestExecutionContext.
+        // The TestExecutionContext class would have to be modified
+        // in order to allow chaining of contexts across a 
+        // remoting boundary.
```

Comment:
@CharliePoole do you think this is something that we should continue to support? Should an issue be created?